modules/configuring-monitoring-for-the-single-model-serving-platform.adoc

```diff
@@ -6,10 +6,14 @@
 [role="_abstract"]
 
 ifdef::self-managed,cloud-service[]
-The single-model serving platform includes metrics for link:{rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#about-the-single-model-serving-platform_serving-large-models[supported runtimes]. You can also configure monitoring for OpenShift Service Mesh. The service mesh metrics helps you to understand dependencies and traffic flow between components in the mesh. When you have configured monitoring, you can grant Prometheus access to scrape the available metrics. 
+The single-model serving platform includes metrics for link:{rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#about-the-single-model-serving-platform_serving-large-models[supported runtimes] of the KServe component. KServe does not generate its own metrics and relies on the underlying serving runtime to provide them. The set of available metrics for a deployed model depends on the runtime that you used.
```

Comment:
Per [this comment](https://github.com/opendatahub-io/opendatahub-documentation/pull/255#discussion_r1572273753), I would say **runtimes**, plural. 

That is:

"KServe does not generate its own metrics and relies on the underlying serving **runtimes** to provide them."